src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -224,6 +226,14 @@ public bool ExecuteReplicationOnce()
 
         private void AddReplicationItemToBatch(ReplicationBatchDocumentItem item)
         {
+            if (item.Flags.HasFlag(DocumentFlags.Artificial))
```

Comment:
Use:
`if ((item.Flags & DocumentFlags.Artificial) == DocumentFlags.Artificial)`

To avoid boxing